client/package.json

```diff
@@ -9,15 +9,20 @@
   "dependencies": {
     "apollo-boost": "^0.4.2",
     "apollo-client": "^2.6.2",
+    "dotenv": "^8.0.0",
```

Comment:
you will need dotenv on the server too. Please install that package on the server.